mmaction/models/losses/cross_entropy_loss.py

```diff
@@ -9,6 +9,17 @@
 class CrossEntropyLoss(BaseWeightedLoss):
     """Cross Entropy Loss.
 
+    Support two kinds of labels and their corresponding loss type. It's worth
+    mentioning that loss type will be detected by the shape of ``cls_score``
+    and ``label``.
+    1) Hard label: This label is an integer array and all of the elements are
+        in the range [0, num_classes - 1]. This label's shape should be
+        ``cls_score``'s shape with the `num_classes` dimension removed.
+    2) Soft label(one-hot like label): This label is a probability distribution
```

Comment:
~~(one-hot like label)~~ (using probablity distribution over classes as label)

`One-hot like` gives more confusion than clarification. If something resembles one-hot, maybe it is "Two-hot" or "One-hotter/cooler"?
"The label shape is the similar to one-hot encoding" is the last thing one may think of